### PR TITLE
POWR-126 Install & enable swiftype module

### PR DESCRIPTION
**Pull request by RichardDavies**

### CODE REVIEW

**Comment by RichardDavies** [comment]
@joshuami Ok, this is ready for another review and I've got a good feeling about it this time :-)

File: web/sites/default/config/swiftype_integration.settings.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+swiftype_integration_install_key: K2y8KFxi3pKyFfU1-csZ
```
Comment (joshuami) [inline web/sites/default/config/swiftype_integration.settings.yml]: @RichardDavies, is there any concern with these keys being published publicly? Are they secrets or just configuration information?

**Comment by RichardDavies** [comment]
Oh, good question. Yeah, we need to keep that secret. Should I remove it from the config and push up another commit?

**Comment by joshuami** [comment]
We'll need to reset those credentials as well. Anything we publish to github, even if we remove it, is essentially compromised because there is no way to know if they were scraped by tools that look for this sort of thing.

**Pull request merged**